cmd/process-agent/app/version.go

```diff
@@ -0,0 +1,44 @@
+package app
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+	"runtime"
+
+	"github.com/DataDog/datadog-agent/pkg/serializer"
+	"github.com/DataDog/datadog-agent/pkg/version"
+)
+
+// VersionCmd is a command that prints the process-agent version data
+var VersionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version info",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return WriteVersion(os.Stdout)
+	},
+	SilenceUsage: true,
```

Comment:
Is there a specific reason to use `SilenceUsage` here? Don't we show errors if they come up?

Reply:
`SilenceUsage` doesn't suppress errors, it just makes sure that the help text isn't displayed in addition to any errors that might occur. However, upon testing I realize that it inherits this value from the parent command, so including it is a  no-op.